docs/links.md

```diff
@@ -82,17 +88,18 @@
 ### Community
 
 - [Black MIDI Community Hub](https://discord.gg/s42aft8) - Main, global community
+- [Creative Society](https://discord.gg/2jb7hTymAH) - Alterative to the BMC. Dedicated to not only Black MIDI, but other artistic skills
 - [黑乐谱活动社区 Black MIDI Score Activity Community](https://discord.gg/m9q2QCAcdN) - China-based community
 - [Black MIDI Indonesia Community [BMIC] 🇮🇩](https://discord.gg/3VXrmnQUTR) - Indonesia-based community
 
 ### Programs
 
-- [Arduano Dev Server](https://discord.gg/8e5FXB) - Zenith, Kiva
+- [Arduano Dev Server](https://discord.gg/8e5FXB) - Zenith, Kiva, Wasabi
 - [BetterTeam Dev Server](https://discord.gg/wwSkpN5) - BetterPFA
 	- [BetterPFA Archive](https://discord.gg/5sMugcQ)
 - [CJC Apps Discussion](https://discord.gg/uxCqAEz) - CJC Toolbox, CJCAMM, CJCOR, CJCMCG
 - [Embers](https://discord.gg/MaCxW5u) - Embers
-- [Keppy's Software](https://discord.gg/73DVswT) - KMC, OmniMIDI 
+- ~~[Keppy's Software](https://discord.gg/73DVswT) - KMC, OmniMIDI~~ [Server no longer exist]
```

Comment:
Please just remove it instead. Last time I also remove unavailable servers but it seems I missed some.


```suggestion
```